src/Castle.Windsor/Castle.Windsor-VS2017.csproj

```diff
@@ -35,4 +35,10 @@
   <PropertyGroup Condition="'$(TargetFramework)'=='net45'">
     <DefineConstants>$(DefineConstants);FEATURE_PERFCOUNTERS;FEATURE_GAC</DefineConstants>
   </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)'=='netstandard1.5'">
```

Comment:
@eerhardt The targets are not set because this is work in progress and the project does not build on core yet. The targets should be `<TargetFrameworks>net45;netstandard1.6</TargetFrameworks>`
I have a version that builds with the right targets at https://github.com/alinapopa/Windsor/tree/netcoreAll
Also the `netstandard1.5` condition is there because I initially planned the target to be netstandard1.5; if the `DependencyModel` will be used, the target will be changed to 1.6.